src/tests/LoggingTest.cpp

```diff
@@ -52,34 +52,37 @@ struct LoggingTest : Cr::TestSuite::Tester {
 
 constexpr const struct {
   const char* envString;
-  const char* expected;
+  const char* dfltDebug;
+  const char* dfltWarning;
```

Comment:
```suggestion
  const char* defaultDebug;
  const char* defaultWarning;
```

Ws knd hrd t rd otherwise :sweat_smile: